e2e/support/github/run-e2e-docker-env.sh

```diff
@@ -29,7 +29,7 @@ echo "Creating dynamic spa-assemble-config.json..."
 jq -n \
   --arg apps "$apps" \
   --arg app_names "$(echo ${app_names[@]})" \
-  '{"@openmrs/esm-primary-navigation-app": "next", "@openmrs/esm-home-app": "next"} + (
+  '{"@openmrs/esm-primary-navigation-app": "next", "@openmrs/esm-laboratory-app": "next", "@openmrs/esm-home-app": "next"} + (
```

Comment:
Let's move the tests that depend on the laboratory app to the laboratory app; we can add the chart there.